core/record.go

```diff
@@ -56,6 +56,43 @@ func (t *Timetrace) ListRecords(date time.Time) ([]*Record, error) {
 	return records, nil
 }
 
+// GetLatestRecord loads and returns the latest record made. If no records
+// are found, an error will be returned.
+func (t *Timetrace) GetLatestRecord() (*Record, error) {
```

Comment:
Nice, but there already is `Timetrace.loadLatestRecord`, which looks pretty much the same 🤔

Reply:
Oh, you're right! I completely overlooked that one :0 
Would you propose to export `loadLatestRecord` and use it in the edit-method or rather stay with `GetLatestRecord` as "interface" and just use `loadLatestRecord` inside?

Reply:
I think it would be ok to just export `loadLatestRecord`.

Reply:
okay, just did that :)